README.md

```diff
@@ -410,32 +422,65 @@ CameraPreview.setExposureCompensation(3);
 <info>Change the size of the preview window.</info><br/>
 
 ```javascript
-CameraPreview.setPreviewSize({width: window.screen.width, height: window.screen.height});
+CameraPreview.setPreviewSize({
+  width: window.screen.width,
+  height: window.screen.height
+});
 ```
 
 ### getSupportedPictureSizes(cb, [errorCallback])
 
 ```javascript
-CameraPreview.getSupportedPictureSizes(function(dimensions){
+CameraPreview.getSupportedPictureSizes(function(dimensions) {
   // note that the portrait version, width and height swapped, of these dimensions are also supported
   dimensions.forEach(function(dimension) {
-    console.log(dimension.width + 'x' + dimension.height);
+    console.log(dimension.width + "x" + dimension.height);
   });
 });
 ```
 
 ### getCameraCharacteristics(cb, [errorCallback])
 
-*Currently this feature is for Android only. A PR for iOS support would be happily accepted*
+_Currently this feature is for Android only. A PR for iOS support would be happily accepted_
 
 <info>Get the characteristics of all available cameras. Returns a JSON object representing the characteristics of all available cameras.</info><br/>
 
 ```javascript
-CameraPreview.getCameraCharacteristics(function(characteristics){
+CameraPreview.getCameraCharacteristics(function(characteristics) {
   console.log(characteristics);
 });
 ```
 
+### backgroundVideoStart(cb, [errorCallback])
+
+_Currently this feature is for Android only._
+
+<info>Record Video and start saving to the cache</info><br/>
+
+```javascript
+CameraPreview.backgroundVideoStart(function(result) {
+  console.log(result);
+});
+```
+
+### backgroundVideoStop(cb, [errorCallback])
+
+_Currently this feature is for Android only._
```

Comment:
Is this actually android only? I am pretty sure the iOS is implemented too.

Reply:
Was iOS ever confirmed?

Reply:
No it wasnt actually implemented. PR wanted for iOS support.